lightningd/peer_control.c

```diff
@@ -1034,7 +1034,7 @@ AUTODATA(json_command, &listpeers_command);
 static struct command_result *
 command_find_channel(struct command *cmd,
 		     const char *buffer, const jsmntok_t *tok,
-		     struct channel **channel)
+		     struct channel **channel, struct peer **peer_out)
```

Comment:
Doesn't `channel` have a pointer to the associated `peer`?

Reply:
yep, addressed by https://github.com/ElementsProject/lightning/pull/2342/commits/a9e84acf67b75923450c94dd738922f0b1df18a6